modules/ipi-install-validation-checklist-for-nodes.adoc

```diff
@@ -8,19 +8,21 @@
 
 .When using the `provisioning` network
 
-* [ ] NIC1 VLAN is configured for the `provisioning` network. (optional)
-* [ ] NIC1 is PXE-enabled on the provisioner, control plane (master), and worker nodes when using a `provisioning` network. (optional)
+* [ ] NIC1 VLAN is configured for the `provisioning` network.
+* [ ] NIC1 for the `provisioning` network is PXE-enabled on the provisioner, control plane (master), and worker nodes.
 * [ ] NIC2 VLAN is configured for the `baremetal` network.
 * [ ] PXE has been disabled on all other NICs.
+* [ ] DNS is configured with API and internal API endpoints.
```

Comment:
As discussed on our call, I think this should be "DNS is configured with API and ingress endpoints."